inspirehep/modules/workflows/tasks/actions.py

```diff
@@ -263,12 +264,15 @@ def refextract(obj, eng):
     pdf_references, text_references = [], []
     source = get_value(obj.data, 'acquisition_source.source')
 
-    pdf = get_pdf_in_workflow(obj)
-    if pdf:
+    tmp_pdf = get_pdf_in_workflow(obj)
```

Comment:
Why this?

Reply:
to make explicit that the pdf that you expect to get is a temporary one (and you'll have to clean it up).